openlibrary/macros/databarAuthor.html

```diff
@@ -74,12 +76,11 @@ <h2>$_('Create a new list')</h2>
         <!-- FOR each list -->
         <li>
             <span class="image">
-            $ cover = "//covers.openlibrary.org/b/olid/%s-M.jpg" % page.cover_edition_key if page.cover_edition_key else "https://openlibrary.org/images/icons/avatar_book-sm.png"
-            <a href="$page.url"><img src="$cover" width="32" alt="$_('Cover of: %(title)s', title=f'{page.title}:{page.subtitle} if page.get('subtitle', None) else page.title)" title="$_('Cover of: %(title)s', title=f'{page.title}:{page.subtitle} if page.get('subtitle', None) else page.title)"/></a>
+            <a href="$page.url"><img src="$cover" width="32" alt="$_('Cover of: %(title)s', title=title)" title="$_('Cover of: %(title)s', title=title)"/></a>
```

Comment:
Woohoo!! so nice so clean!

Reply:
oops, I did find one mistake like 12 `('(%(page)s, page=$page)` -> `('(%(page)s, page=page)`. 
Corrected for next commit !